### DISCUSSION TITLE
Complicated Select and Multiselect

### DISCUSSION THREAD

**Discussion by Kuuotai** [taiga-family/taiga-ui#3163] [General]
Why is TuiSelect and MultiSelect so complicated?? The whole stringify pure function and the **ItemContent** vs **ValueContent**  is very annoying and just bloated code for us to right. There has to be way simpler way to use tuiselect and multiselect

**Comment by loxy** [comment]
Because it is so flexible. The best implementation I've ever worked with. Just create a simpler wrapper component if you want a simple API which fits your needs.

**Comment by waterplea** [comment]
Depending on the complexity of your actual data, workflow and templates it can be as simple as:
```html
<tui-select [(ngModel)]>
  <tui-data-list-wrapper
    *tuiDataList
    [items]="items"
  </tui-data-list-wrapper>
</tui-select>
```
If you have a sophisticated case sure it would look more complex, but current implementation gives nearly limitless freedom and we need to cover a lot of use cases.
